packages/@aws-cdk/aws-kinesisfirehose/lib/delivery-stream.ts

```diff
@@ -183,6 +204,22 @@ export interface DeliveryStreamProps {
    * @default - a role will be created with default permissions.
    */
   readonly role?: iam.IRole;
+
+  /**
+   * Indicates the type of customer master key (CMK) to use for server-side encryption, if any.
+   *
+   * If `encryptionKey` is provided, this will be implicitly set to `CUSTOMER_MANAGED`.
```

Comment:
This information doesn't quite match the implementation, which throws an error if the key is provided, but the encryption type is anything other than `CUSTOMER_MANAGED`.

Reply:
The implementation will not throw an error if the key is provided, and encryptionType is undefined.